projects/example-app/src/app/books/containers/find-book-page.component.ts

```diff
@@ -13,9 +13,9 @@ import * as fromBooks from '@example-app/books/reducers';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <bc-book-search
-      [query]="searchQuery$ | async"
-      [searching]="loading$ | async"
-      [error]="error$ | async"
+      [query]="(searchQuery$ | async) ?? ''"
+      [searching]="(loading$ | async)!"
+      [error]="(error$ | async)!"
```

Comment:
maybe the `error` should have its type changed to support a falsy value since there may not be an error sometimes